src/main/scala/lms/collection/ArrayOps.scala

```diff
@@ -64,3 +64,35 @@ trait ArrayOps { b: Base =>
 }
 
 // FIXME(feiw) CodeGen of Array is still not yet separated from ExtendedScalaCodeGen and ExtendedCCodeGen
+
+// StackArray is the array using stack (i.e. int a[5] = {1,2,3,4,5};)
+trait StackArrayOps extends ArrayOps { b: Base =>
+
+  def NewStackArray[T:Manifest](x: Rep[Int]): Rep[Array[T]] = {
```

Comment:
Can `NewStackArray` and `StackArray` use the same name?

Reply:
probably not. The parameter in NewStackArray is for size. The parameter for StackArray is for content. There might be ambiguity if they have the same name?